code/__DEFINES/mobs.dm

```diff
@@ -0,0 +1,3 @@
+//Grasp indexes
```

Comment:
Yes, but if you're planning to do that later there's no real point in not just including these into setup.dm